admin/app/views/workarea/admin/pricing_skus/edit.html.haml

```diff
@@ -31,6 +31,8 @@
           = label_tag 'sku[tax_code]', t('workarea.admin.fields.tax_code'), class: 'property__name'
           = text_field_tag 'sku[tax_code]', @sku.tax_code, class: 'text-box'
 
+        = append_partials('admin.add_cost', sku: @sku)
```

Comment:
Also here, looks like we have `admin.pricing_sku_fields` already in place.

Reply:
The Edit form throws me off, I think using the same append point that is used in the New form would be fine as long as the MSRP field is moved below On Sale and Allow Discounting, the same way it is laid out in the New form. Picture below for reference, the proposed changes are both using the existing append points.

<img width="1115" alt="Screen Shot 2020-02-06 at 7 38 51 PM" src="https://user-images.githubusercontent.com/37818357/73993761-d46a4b00-4918-11ea-8b9c-9ef5b9f7fe67.png">